From 1e6a96e9573164bfacb1a2fae16de00763bb06da Mon Sep 17 00:00:00 2001
From: Ryan Barnett <ryanbarnett3@gmail.com>
Date: Sat, 20 Feb 2021 13:40:05 -0600
Subject: [PATCH 1/1] Makefile: seperate manpages and utils install

To allow the opkg-utils to be installed separately from the manpages,
break apart the install step into two install steps: install-utils and
install-docs

Signed-off-by: Ryan Barnett <ryanbarnett3@gmail.com>
---
 Makefile | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 4049654..7b7b8d5 100644
--- a/Makefile
+++ b/Makefile
@@ -25,11 +25,13 @@ mandir ?= $(PREFIX)/man
 %.1: %
 	pod2man -r "" -c "opkg-utils Documentation" $< $@
 
-all: $(UTILS) $(MANPAGES)
+all: install
 
-install: all
+install-utils: $(UTILS)
 	install -d $(DESTDIR)$(bindir)
 	install -m 755 $(UTILS) $(DESTDIR)$(bindir)
+
+install-docs: $(MANPAGES)
 	install -d $(DESTDIR)$(mandir)
 	for m in $(MANPAGES); \
 	do \
@@ -37,4 +39,6 @@ install: all
 		install -m 644 "$$m" $(DESTDIR)$(mandir)/man$${m##*.}; \
 	done
 
-.PHONY: install all
+install: install-utils install-docs
+
+.PHONY: install install-utils install-docs all
-- 
2.25.1

